packages/server/src/automations/triggers.ts

```diff
@@ -94,7 +94,7 @@ export async function externalTrigger(
   automation: Automation,
   params: { fields: Record<string, any>; timeout?: number },
   { getResponses }: { getResponses?: boolean } = {}
-) {
+): Promise<any> {
```

Comment:
Would be good to type this one - but the whole async/sync thing does make it a lot more difficult